Elsewhen/Elements/DateTimeSelection.swift

```diff
@@ -58,7 +58,7 @@ struct DateTimeSelection: View, OrientationObserving {
         mediumImpactFeedbackGenerator.impactOccurred()
 #endif
         self.selectedDate = Date()
-        self.selectedTimeZone = TimeZone.current
+        self.selectedTimeZone = UserDefaults.shared.resetButtonTimeZone
```

Comment:
Is there a reason the default (`TimeZone.current`) isn't assigned here?

Reply:
It largely doesn't matter, but because the lower views that need it always coerce `nil` into the current timezone, so it just seemed to make more logical sense to "default to [what the user has set as the default]" rather than "where they currently are" ?

Reply:
ah, I didn't explain myself very well. Why not do it like this?

`self.selectedTimeZone = UserDefaults.shared.resetButtonTimeZone ?? TimeZone.current` rather than coercing nil every time we use `self.selectedTimeZone`?

Reply:
Ah okay! Mostly because I needed it to be an Optional so that places that need to can differentiate between "no time zone set: use TimeZone.current" and "set to Europe/London". This allows the "use TimeZone.current" conversion to happen at the last minute possible so that other code knows the difference

Reply:
I feel like there's probably a neater way to do this, but that makes sense 🙂